apps/address-book/arapp.json

```diff
@@ -17,15 +17,21 @@
       "network": "rpc",
       "appName": "address-book.aragonpm.eth"
     },
-    "rinkeby": {
+    "staging": {
       "registry": "0x98df287b6c145399aaa709692c8d308357bc085d",
-      "appName": "tps-address-book.open.aragonpm.eth",
```

Comment:
I hope we track this since we have the apps published on rinkeby environment, why was it needed to wipe this environment that is currently being used?